documentation/modules/exploit/linux/http/spring_cloud_gateway_rce.md

```diff
@@ -11,7 +11,7 @@ the versions affected are:
 A sample demo [project](https://github.com/wdahlenburg/spring-gateway-demo) is available,
 which can be used to run a vulnerable server by following the installation instructions below.
     
-### Installation Instructions
+## Installation Instructions
```

Comment:
Previous variant was valid, this is technically not a header that msftidy_docs.rb recognizes so it must be placed as a H3 or lower header. No H1 headers are allowed according to msftidy_docs.rb.

Reply:
Fixed